kokoro/build_binaries_rust.sh

```diff
@@ -42,6 +43,7 @@ readonly generated_binaries=(
 )
 readonly binary_names=(
     oak_restricted_kernel_simple_io_wrapper_bin
+    oak_restricted_kernel_simple_io_init_rd_wrapper_bin
```

Comment:
I assume the new one would also behave like a bzImage (i.e. a kernel-type endorsement) since it is named "wrapper".

Reply:
Yes